routing/pathfind_test.go

```diff
@@ -620,56 +628,86 @@ type expectedHop struct {
 }
 
 type basicGraphPathFindingTestCase struct {
+	name                  string
 	target                string
 	paymentAmt            btcutil.Amount
-	feeLimit              lnwire.MilliSatoshi
+	pathRestrictions      PathRestrictions
 	expectedTotalAmt      lnwire.MilliSatoshi
 	expectedTotalTimeLock uint32
 	expectedHops          []expectedHop
-	expectFailureNoPath   bool
+	expectError           bool
+	expectedFailure       errorCode
 }
 
 var basicGraphPathFindingTests = []basicGraphPathFindingTestCase{
 	// Basic route with one intermediate hop.
-	{target: "sophon", paymentAmt: 100, feeLimit: noFeeLimit,
-		expectedTotalTimeLock: 102, expectedTotalAmt: 100110,
+	{name: "BasicRouteIntermediateHop ", target: "sophon", paymentAmt: 100,
+		pathRestrictions:      PathRestrictions{FeeLimit: noFeeLimit, MaxCltvDelay: noCltvLimit},
+		expectedTotalTimeLock: 245, expectedTotalAmt: 100110,
 		expectedHops: []expectedHop{
 			{alias: "songoku", fwdAmount: 100000, fee: 110, timeLock: 101},
 			{alias: "sophon", fwdAmount: 100000, fee: 0, timeLock: 101},
 		}},
 
 	// Basic direct (one hop) route.
-	{target: "luoji", paymentAmt: 100, feeLimit: noFeeLimit,
+	{name: "BasicRouteDirect", target: "luoji", paymentAmt: 100,
```

Comment:
It's a bit hard to see the changing parameters now. I think it would be easier to parse of we did it like 
```go
{
    name: "BasicRouteDirect", 
    target: "luoji", 
    paymentAmt: 100,
...
}.
```